### ISSUE TITLE
Process exceeded running time limit even if I changed the config. file

### ISSUE THREAD

**Issue by andrewucla** [nf-core/sarek#711]
### Description of the bug

I am running the WES for my tumor samples and I am getting the 16h time limit for manta. Then I changed the config file according to the suggestions on this webpage https://nf-co.re/sarek/3.0.1/usage, but the problem persisted.

### Command used and terminal output

```console
1. The slurm file I am running
#!/bin/bash
#SBATCH -o PDO106.out
#SBATCH -e PDO106.err
#SBATCH -J PDO106.job
#SBATCH -n 1
#SBATCH -N 1
#SBATCH -c 16
#SBATCH -t 200:00:00
#SBATCH --mem=64G
#SBATCH --get-user-env
#SBATCH --mail-type=END,FAIL
#SBATCH --mail-user=ew152@duke.edu

nextflow run nf-core/sarek --max_memory 64.GB -profile singularity --wes --intervals /work/ew152/WES/hg38_bed3_GATK.bed --input PDO106.csv --outdir . --tools manta,snpeff -c /work/ew152/WES/01.RawData/WES.conf

2. The customized config files WES.conf
process {
      withName:NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MANTA_TUMORONLY:MANTA_TUMORONLY {
          memory = 64.GB
          time = 500.h
    }
      withName:NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_STRELKA_SINGLE:STRELKA_SINGLE {
          memory = 64.GB
          time = 500.h
    }
      withName:NFCORE_SAREK:SAREK:FASTP {
          memory = 64.GB
          time = 500.h
    }
}

3. the error I am still getting
executor >  local (236)
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[1f/bd0c04] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[b3/e1eb24] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 1 of 1 ✔
[49/95183c] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 65 of 65 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[6a/042598] process > NFCORE_SAREK:SAREK:RUN_FAST... [100%] 2 of 2 ✔
[45/7e82c7] process > NFCORE_SAREK:SAREK:FASTP (P... [100%] 2 of 2 ✔
[c8/e3793d] process > NFCORE_SAREK:SAREK:GATK4_MA... [100%] 24 of 24 ✔
[-        ] process > NFCORE_SAREK:SAREK:GATK4_MA... -
[-        ] process > NFCORE_SAREK:SAREK:GATK4_MA... -
[22/5a95fd] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[79/474c72] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[51/51b113] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[cc/fd019f] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[dc/856539] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 65 of 65 ✔
[72/e05db9] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 1 of 1 ✔
[30/dce7e6] process > NFCORE_SAREK:SAREK:RECALIBR... [100%] 65 of 65 ✔
[c5/4fbe6d] process > NFCORE_SAREK:SAREK:RECALIBR... [100%] 1 of 1 ✔
[80/15783c] process > NFCORE_SAREK:SAREK:RECALIBR... [100%] 1 of 1 ✔
[7c/78f8c8] process > NFCORE_SAREK:SAREK:CRAM_QC:... [100%] 1 of 1 ✔
[35/23b3fb] process > NFCORE_SAREK:SAREK:CRAM_QC:... [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:SAMTOOLS... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[5a/71125b] process > NFCORE_SAREK:SAREK:TUMOR_ON... [  1%] 1 of 65, failed: 1
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:B... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:V... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:V... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:V... -
[-        ] process > NFCORE_SAREK:SAREK:ANNOTATE... -
[-        ] process > NFCORE_SAREK:SAREK:ANNOTATE... -
[-        ] process > NFCORE_SAREK:SAREK:CUSTOM_D... -
[-        ] process > NFCORE_SAREK:SAREK:MULTIQC     -
Execution cancelled -- Finishing pending tasks before exit
Error executing process > 'NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MANTA_TUMORONLY:MANTA_TUMORONLY (PDO)'

Caused by:
  Process exceeded running time limit (16h)

Command executed:

  configManta.py         --tumorBam PDO.recal.cram         --reference Homo_sapiens_assembly38.fasta         --exome --callRegions chr17_17137915-17138077.bed.gz         --runDir manta

  python manta/runWorkflow.py -m local -j 12

  mv manta/results/variants/candidateSmallIndels.vcf.gz         PDO.manta.chr17_17137915-17138077.candidate_small_indels.vcf.gz
  mv manta/results/variants/candidateSmallIndels.vcf.gz.tbi         PDO.manta.chr17_17137915-17138077.candidate_small_indels.vcf.gz.tbi
  mv manta/results/variants/candidateSV.vcf.gz         PDO.manta.chr17_17137915-17138077.candidate_sv.vcf.gz
  mv manta/results/variants/candidateSV.vcf.gz.tbi         PDO.manta.chr17_17137915-17138077.candidate_sv.vcf.gz.tbi
  mv manta/results/variants/tumorSV.vcf.gz         PDO.manta.chr17_17137915-17138077.tumor_sv.vcf.gz
  mv manta/results/variants/tumorSV.vcf.gz.tbi         PDO.manta.chr17_17137915-17138077.tumor_sv.vcf.gz.tbi

  cat <<-END_VERSIONS > versions.yml
  "NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MANTA_TUMORONLY:MANTA_TUMORONLY":
      manta: $( configManta.py --version )
  END_VERSIONS

Command exit status:
  -

Command output:

  Successfully created workflow run script.
  To execute the workflow, run the following script and set appropriate options:

  manta/runWorkflow.py

Command error:
  INFO:    Converting SIF file to temporary sandbox...
  .command.run: line 31: /dev/fd/62: No such file or directory

Work dir:
  /work/ew152/WES/01.RawData/106_out/work/5a/71125b027320b6fc97a79fbbdfeb9c

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line
```

### System information

nextflow                  22.04.0              h4a94de4_0    bioconda
nf-core                   2.4.1              pyh5e36f6f_1    bioconda

Running slurm on the HPC, with the Sigularity container.

**Comment by FriederikeHanssen** [comment]
Hi! Can you try quoting the process names? 
```
withName: 'NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MANTA_TUMORONLY:MANTA_TUMORONLY' {
}
```

**Comment by andrewucla** [comment]
> Hi! Can you try quoting the process names?
> 
> ```
> withName: 'NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MANTA_TUMORONLY:MANTA_TUMORONLY' {
> }
> ```

**Ohhhh. My bad. Thank you so much for the prompt response. I will give it a try.

Also, in my other run with the command**

#!/bin/bash
#SBATCH -o PDOX106.out
#SBATCH -e PDOX106.err
#SBATCH -J PDOX106.job
#SBATCH -n 1
#SBATCH -N 1
#SBATCH -c 16
#SBATCH -t 100:00:00
#SBATCH --mem=64G
#SBATCH --get-user-env
#SBATCH --mail-type=END,FAIL
#SBATCH --mail-user=ew152@duke.edu

nextflow run nf-core/sarek --max_memory 64.GB -profile singularity --wes --intervals /work/ew152/WES/hg38_bed3_GATK.bed --input PDOX106.csv --outdir . --tools strelka,manta,controlfreec,vep,snpeff -c /work/ew152/WES/01.RawData/WES.conf -resume

**I am getting the following error:**
executor >  local (240)
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[3f/4ffd98] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[ca/37f118] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[d2/7f948a] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 1 of 1 ✔
[44/b6851d] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 65 of 65 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMEN... -
[ab/6e21c5] process > NFCORE_SAREK:SAREK:RUN_FAST... [100%] 2 of 2 ✔
[9f/5aafd8] process > NFCORE_SAREK:SAREK:FASTP (P... [100%] 2 of 2 ✔
[5c/934b59] process > NFCORE_SAREK:SAREK:GATK4_MA... [100%] 24 of 24 ✔
[-        ] process > NFCORE_SAREK:SAREK:GATK4_MA... -
[-        ] process > NFCORE_SAREK:SAREK:GATK4_MA... -
[7b/57e8d7] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[ae/f04cb6] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[98/72d077] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[f1/b40fe3] process > NFCORE_SAREK:SAREK:MARKDUPL... [100%] 1 of 1 ✔
[bc/74dd9d] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 65 of 65 ✔
[19/9d9cc2] process > NFCORE_SAREK:SAREK:PREPARE_... [100%] 1 of 1 ✔
[30/100b3f] process > NFCORE_SAREK:SAREK:RECALIBR... [100%] 65 of 65 ✔
[85/78744f] process > NFCORE_SAREK:SAREK:RECALIBR... [100%] 1 of 1 ✔
[92/ee2242] process > NFCORE_SAREK:SAREK:RECALIBR... [100%] 1 of 1 ✔
[d9/0f34e7] process > NFCORE_SAREK:SAREK:CRAM_QC:... [100%] 1 of 1 ✔
[f3/d4efff] process > NFCORE_SAREK:SAREK:CRAM_QC:... [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:SAMTOOLS... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[-        ] process > NFCORE_SAREK:SAREK:GERMLINE... -
[7f/76b398] process > NFCORE_SAREK:SAREK:TUMOR_ON... [  7%] 5 of 65, failed: 1
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... [  0%] 0 of 65
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... [  0%] 0 of 65
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:TUMOR_ON... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:PAIR_VAR... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:B... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:V... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:V... -
[-        ] process > NFCORE_SAREK:SAREK:VCF_QC:V... -
[-        ] process > NFCORE_SAREK:SAREK:ANNOTATE... -
[-        ] process > NFCORE_SAREK:SAREK:ANNOTATE... -
[-        ] process > NFCORE_SAREK:SAREK:ANNOTATE... -
[-        ] process > NFCORE_SAREK:SAREK:ANNOTATE... -
[-        ] process > NFCORE_SAREK:SAREK:CUSTOM_D... -
[-        ] process > NFCORE_SAREK:SAREK:MULTIQC     -
-[nf-core/sarek] Pipeline completed with errors-
Error executing process > 'NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MPILEUP:SAMTOOLS_MPILEUP (PDOX)'

Caused by:
  Process `NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MPILEUP:SAMTOOLS_MPILEUP (PDOX)` terminated with an error exit status (1)

Command executed:

  samtools mpileup \
      --fasta-ref Homo_sapiens_assembly38.fasta \
      --output PDOX_chr8_130361670-130361707.tumor.mpileup \
       \
      PDOX.recal.cram
  bgzip PDOX_chr8_130361670-130361707.tumor.mpileup
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MPILEUP:SAMTOOLS_MPILEUP":
      samtools: $(echo $(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*$//')
  END_VERSIONS

Command exit status:
  1

Command output:
  (empty)

Command error:
  INFO:    Converting SIF file to temporary sandbox...
  [mpileup] 1 samples in 1 input files
  .command.run: line 31: /dev/fd/62: No such file or directory
  bgzip: error while loading shared libraries: libdeflate.so.0: cannot open shared object file: No such file or directory
  .command.run: line 155: kill: (3047177) - No such process
  INFO:    Cleaning up image...

Work dir:
  /work/ew152/WES/OX106/work/53/16022563366df8e1f516936963679c

Tip: you can replicate the issue by changing to the process work

**Do you know what is going on here?**

**Comment by FriederikeHanssen** [comment]
Hi! No not sure what is going on on one glance. however, we released yesterday a patch for mpileup with 3.0.1. I would highly recommend switching to that version if you want to use controlfreec or mpileup :)

**Comment by FriederikeHanssen** [comment]
Is this working now? I would close the issue since the original problem is solved. feel free to open a new issue otherwise :)
